language/move-prover/spec-lang/src/env.rs

```diff
@@ -515,6 +515,7 @@ impl<'env> ModuleEnv<'env> {
                     self.env
                         .source_files
                         .name(self.data.source_file_id)
+                        .to_string_lossy()
```

Comment:
What's the `lossy` here?

Reply:
The file names in 0.8.0 are no longer `String`s but `OsString`s, so we must convert it to unicode here. I used `to_string_lossy()` since that one is infallible.